content/en/docs/tasks/observability/metrics/customize-metrics/index.md

```diff
@@ -71,7 +71,7 @@ the example application throughout this task.
     To customize telemetry v2 metrics, for example, to add `request_host`
     and `destination_port` dimensions to the `requests_total` metric emitted by both
     gateways and sidecars in the inbound and outbound direction, change the installation
-    options as follows:
+    options with command [istioctl install -f](/docs/setup/install/istioctl/#install-istio-using-the-default-profile) and yaml as follows:
```

Comment:
may be you can add the command in the below section along with the yaml in {{text bash}} ?